core/blockchain.go

```diff
@@ -1053,9 +1055,47 @@ func (bc *BlockChain) Accept(block *types.Block) error {
 	bc.addAcceptorQueue(block)
 	acceptedBlockGasUsedCounter.Inc(int64(block.GasUsed()))
 	acceptedTxsCounter.Inc(int64(len(block.Transactions())))
+	if baseFee := block.BaseFee(); baseFee != nil {
+		blockGasPriceGauge.Update(baseFee.Int64())
+	}
+	total, err := TotalFees(block, bc.GetReceiptsByHash(block.Hash()))
+	if err != nil {
+		log.Debug(fmt.Sprintf("TotalFees %s", err))
```

Comment:
Can we change this to `Error` level? This error should never happen since the block has been accepted and we should have the same number of transactions and receipts for the indexed block.